tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "We now need to make sure that all the hydrogens are added to our protein. This process will also generate the\n",
+    "parameter/topology file we need.\n",
+    "\n",
+    "`% gmx pdb2gmx -f protein.pdb -ignh -o protein.gro`\n",
+    "\n",
+    "The program should run and present a list of force-fields from which to select. Select the AMBER99SB-ILDN force field which should be option 6 in the list followed by 1 to select the recommended TIP3P water. If all goes well this should generate several files:\n",
+    "1. topol.top \n",
+    "2. topol_Protein_chain_A.itp \n",
+    "3. topol_Protein_chain_B.itp \n",
+    "3. posre_Protein_chain_A.itp\n",
+    "4. posre_Protein_chain_B.itp\n",
+    "5. protein.gro\n",
+    "\n",
+    "Type:\n",
+    "\n",
+    "`% ls`\n",
+    "\n",
+    "to verify that all the above files have been created and are in your directory.\\\n",
+    "Note that the protein has a net charge of +4e. You should see a line that says \"Total charge\n",
+    "in system 4.000 e\".\n",
+    "\n",
+    "Before we can add water we need to define a box in which to put the protein and the water:\n",
+    "\n",
+    "`% gmx editconf -f protein.gro -box 7 7 7 -c -o boxed.gro`\n",
+    "\n",
+    "This puts the protein in the centre of the box that is 7 nm x 7 nm x 7 nm and creates the resulting file boxed.pdb .\\\n",
+    "Next we need to add water to the system. We will also add ions -enough to neutralize the system and to a\n",
+    "concentration that is representative of the cell. We can add the water by repeatedly overlaying a small box of water into the system (216 molecules).\n",
+    "\n",
+    "`% gmx solvate -cp boxed.gro -cs -o solvated.gro -p topol.top`\n",
+    "\n",
+    "You may have noticed in some of the output generated that the total system charge is +4. In order for us to use an Ewald\n",
+    "method to calculate the electrostatic interactions we need to have a neutral system overall. Therefore we will add\n",
+    "counterions (chloride ions, in this case) using the option -neutral and enough ions to make the solution up to 150 mM (-conc 0.15). This is done by replacing random water molecules (SOL) with NA+ or CL- ions.\n",
+    "\n",
+    "`% gmx grompp -c solvated.gro -p topol.top -f /home/shared/data/genion.mdp -o genion.tpr`\\\n",
+    "`% gmx genion -s genion.tpr -conc 0.15 -neutral -pname NA -nname CL -o system.gro -p topol.top`\n",
+    "\n",
+    "When prompted, enter the group that corresponds to SOL (should be 13 or thereabouts).\n",
+    "\n",
+    "### Energy Minimization\n",
+    "\n",
+    "Before we can run the actual dynamics, we need to first minimize the energy of the system. Ideally you would minimize down until the forces were below a certain level (tolerance), but we will just give a quick burst of 200 steps here. Since we have finished setting up the system, we will now move to a new directory to perform our simulation from:\n",
```

Comment:
It might be good to add a line here to mention why we do this (i.e. maybe mention that water & hydrogen addition is never ideal, neither are input crystal structures).

Reply:
An extra sentence explaining the need for EM added.